lib/server/sse_handler.dart

```diff
@@ -17,7 +17,7 @@ String _sseHeaders(String origin) => 'HTTP/1.1 200 OK\r\n'
     'Connection: keep-alive\r\n'
     'Access-Control-Allow-Credentials: true\r\n'
     'Access-Control-Allow-Origin: $origin\r\n'
```

Comment:
should these be consistent?

Reply:
cc @liuming0, what's the impact of changing these all to `\n\n`?

Reply:
Note that making them consistent does not impact local tests.